crowbar_framework/config/routes.rb

```diff
@@ -81,14 +81,15 @@
   # barclamps
   get "crowbar/:controller/1.0/help", action: "help", as: "help_barclamp"
   get "crowbar/:controller/1.0/proposals/nodes", action: "nodes", as: "barclamp_nodes"
-  put "crowbar/:controller/1.0/proposals", action: "proposal_create", as: "create_proposal_barclamp"
-  get "crowbar/:controller/1.0/proposals", action: "proposals", as: "proposals_barclamp"
-  post "crowbar/:controller/1.0/proposals/commit/:id", action: "proposal_commit", as: "commit_proposal_barclamp"
+  put "crowbar/:controller/1.0/proposals(.:format)", action: "proposal_create", as: "create_proposal_barclamp"
```

Comment:
Metrics/LineLength: Line is too long. [110/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)